configure.ac

```diff
@@ -22,6 +22,12 @@ AC_DISABLE_STATIC
 AC_PROG_LIBTOOL
 AC_CANONICAL_HOST
 
+# https://github.com/rsyslog/rsyslog/issues/1179
+# Seems libtool doesn't honors ARFLAGS, but statically set AR_FLAGS=cru
+# This causes a few warnings on build:
+#   "ar u modifier ignored since D is the default (see U)"
+m4_divert_text([DEFAULTS], [: "${ARFLAGS=Ucru} ${AR_FLAGS=Ucru}"])
```

Comment:
Judging from the testbench failures, this does not look portable. @Whissi @mbiebl do you have an idea of how this could be addressed?